### PR TITLE
Release 1.4.5 tag fixes

### PR DESCRIPTION
**Pull request by sacherjj**
Updates to fix cargo publish and building for 1.4.5 tag and release.
Removed client, cargo_casper, validation util
Bumped AssemblyScript

### CODE REVIEW

**Review by TomVasile — APPROVED** [review]

File: smart_contracts/contract_as/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "casper-contract",
-  "version": "1.4.4",
+  "version": "1.4.5",
```
Comment (Fraser999) [inline smart_contracts/contract_as/package.json:3]: Was this deliberate?

Comment (sacherjj) [reply smart_contracts/contract_as/package.json:3]: I wanted to make sure it was bumped so that it doesn't block CI.

**Review by Fraser999 — APPROVED** [review]

**Comment by sacherjj** [comment]
bors merge

**Comment by sacherjj** [comment]
bors merge

**Comment by sacherjj** [comment]
bors merge

**Comment by sacherjj** [comment]
bors merge

**Comment by sacherjj** [comment]
bors merge

**Pull request merged**